### PR TITLE
button: don't add "px" to border width

### PR DESCRIPTION
**Pull request by eranshabi**
Like this PR but for `border-width`:
https://github.com/wix/wix-ui-tpa/pull/84

### CODE REVIEW

File: src/components/Button/Button.st.css [hunk 1]
```diff
@@ -110,7 +110,7 @@
     color: value(SecondaryTextColor);
     background-color: wixApply(color, wixApply(fallback, value(MainBackgroundColor), transparent));
     border-color: value(SecondaryBorderColor);
-    border-width: wixApply(unit, wixApply(fallback, wixApply(zeroAsTrue, value(MainBorderWidth)), 1), px);
+    border-width: wixApply(fallback, wixApply(zeroAsTrue, value(MainBorderWidth)), 1);
```
Comment (jonathanadler) [inline src/components/Button/Button.st.css]: the `1` should be with units: `1px` or `wixApply(unit, 1, px)`

File: stories/Button/ButtonExtendedWithStyleParamsExample.st.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  -st-default: TPAButton;
+}
+
+.root {
+  -st-extends: TPAButton;
+  -st-mixin: TPAButton(
+          MainTextColor '"--buttonTextColor"',
+          MainBackgroundColor '"--buttonBackgroundColor"',
+          MainTextFont '"--buttonTextFont"',
+          MainBorderWidth 'unit(fallback(--borderWidth, 1), px)',
```
Comment (jonathanadler) [inline stories/Button/ButtonExtendedWithStyleParamsExample.st.css]: I think the default value should be 0, like in the slider

**Comment by eranshabi** [comment]
fixed

**Comment by jonathanadler** [comment]
> Are there any other places that have units that need to be removed? maybe we can close them all with one pr

@eranshabi 
Lets see if there are any other places we need to remove the `px` unit, and try to combine them all to one pr

**Comment by eranshabi** [comment]
> > Are there any other places that have units that need to be removed? maybe we can close them all with one pr
> 
> @eranshabi
> Lets see if there are any other places we need to remove the `px` unit, and try to combine them all to one pr

that would be all for button

**Pull request merged**